File: xssGuestBook/visit.py
#!/bin python2
#-*- coding:utf-8-*-

import urllib2
import urllib
import cookielib
# import time


class Visit:
    def __init__(self):
        self.cookies = cookielib.CookieJar()
    def admin(self):
        url = 'http://pentest.atomsquare.com/xssGuestBook/admin/_login.php'
        postData = urllib.urlencode(
        {
            'uname':'ddmin',
            'password':'Xsser@lc4t'
        })
        opener = urllib2.build_opener(urllib2.HTTPCookieProcessor(self.cookies))
        request = urllib2.Request(url = url, data = postData)
        result = opener.open(request)
        # print (result.read())

    def delete(self):
        self.admin()
        urldelete = "http://pentest.atomsquare.com/xssGuestBook/admin/__CnSs__ReCovEry.php"
        opener = urllib2.build_opener(urllib2.HTTPCookieProcessor(self.cookies))
        request = urllib2.Request(url = urldelete)
        result = opener.open(request)
        # print (result.read())


import webbrowser
import time
import os

while (True):
    webbrowser.open("http://192.168.1.91/visit.php")
    time.sleep(300)
    delete = Visit()
    delete.delete()
    os.system('taskkill -im chrome.exe')

File: visit.php
<?php
setcookie('flag','CNSS{what_do_you_think_of_xss}');
$cookie_jar = dirname(__FILE__)."/pic.cookie";
$ch = curl_init();
$data = array('uname' => 'ddmin', 'password' => 'Xsser@lc4t');
curl_setopt($ch, CURLOPT_URL, 'http://pentest.atomsquare.com/xssGuestBook/admin/_login.php');
curl_setopt($ch, CURLOPT_POST, 1);
// curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
curl_setopt($ch, CURLOPT_POSTFIELDS, $data);
curl_setopt($ch, CURLOPT_COOKIEFILE, $cookie_jar);
curl_setopt($ch, CURLOPT_HEADER,1);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, false);

$ch = curl_init();
curl_setopt($ch, CURLOPT_URL, 'http://pentest.atomsquare.com/xssGuestBook/admin/_admin.php');

curl_setopt($ch, CURLOPT_POSTFIELDS, $data);
curl_setopt($ch, CURLOPT_COOKIEFILE, $cookie_jar);
// curl_setopt($ch, CURLOPT_HEADER,1);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, false);
print @curl_exec($ch);




?>